UPDATING.md

```diff
@@ -71,6 +71,20 @@ https://developers.google.com/style/inclusive-documentation
 
 -->
 
+### Connexion is now vendored in
+
+The `connexion` used for our REST API depends on `requests` library that's why we had to
+vendor it in (in `airflow/_vendor` package). Airflow does not depend on `connexion` any more.
+This is until `chardet` becomes an optional dependency of `requests`.
+
+See [PR to replace chardet with charset-normalizer](https://github.com/psf/requests/pull/5797)
+
```

Comment:
I don't think this is really relevant to users who are upgrading-- there is nothing they have to do with this information.


```suggestion
```